winner-tool/src/main.rs

```diff
@@ -116,10 +116,25 @@ fn main() {
         })
     };
 
+    // Native loaders do not work when runtime is a git or crates.io dependency
+    let new_bank_callback = {
+        Arc::new(move |bank: &mut Bank| {
+            bank.add_instruction_processor(
```

Comment:
@mvines I'm intercepting instructions for the stake and vote loader programs here. Do we expect other native loaders to be enabled for TdS? Like maybe bpf, budget, or storage?